src/sidebar/components/MenuItem.tsx

```diff
@@ -55,7 +53,7 @@ function SubmenuToggle({
       onClick={onToggleSubmenu}
       title={title}
     >
-      <Icon className="w-3 h-3" />
+      <Icon className="w-3 h-3" viewBox="0 0 16 16" />
```

Comment:
`MenuCollapseIcon` is missing the `viewBox` attribute, which makes it grow outside its container for some reason.

We should probably fix it in frontend-shared, but adding it here works for now, as `MenuExpandIcon` uses the same `viewBox`.